test/rowtransforms.jl

```diff
@@ -0,0 +1,56 @@
+include("imports.jl")
+
+@testset ExtendedTestSet "`NormalizeRow`" begin
+    cols = [:col1, :col2, :col3]
+    item = TabularItem((; zip(cols, [1, "a", 10])...), cols)
```

Comment:
You can save on the splatting by using the NamedTuple constructor:
```suggestion
    item = TabularItem(NamedTuple(zip(cols, [1, "a", 10])), cols)
```